api/app/routers/morecast_v2.py

```diff
@@ -146,3 +150,41 @@ async def get_observed_dailies(start_date: date, end_date: date, request: Observ
                                                                    unique_station_codes)
 
         return ObservedStationDailiesResponse(dailies=observed_dailies)
+
+
+@router.post('/determinates/{start_date}/{end_date}',
+             response_model=IndeterminateDailiesResponse)
+async def get_determinates_for_date_range(start_date: date,
+                                          end_date: date,
+                                          request: StationsRequest):
+    """ Returns the weather values for any actuals, predictions and forecasts for the 
+    requested stations within the requested date range.
+    """
+    logger.info('/morecast-2/determinates/%s/%s', start_date, end_date)
+
+    unique_station_codes = list(set(request.stations))
+
+    vancouver_tz = pytz.timezone("America/Vancouver")
+
+    start_time = vancouver_tz.localize(datetime.combine(start_date, time.min))
+    end_time = vancouver_tz.localize(datetime.combine(end_date, time.max))
+
+    with get_read_session_scope() as db_session:
+        forecasts: List[MoreCastForecastOutput] = get_forecasts(db_session, start_time, end_time, request.stations)
```

Comment:
Do you think it would be possible to reshape forecast objects to match `WeatherIndeterminate`?

Reply:
Oh wait, I guess we're not consuming these on the front end yet, so no need at the moment.

Reply:
Yeah I was gonna hold off on doing that until we start using them. Also there's a couple differences a forecast has currently: no station name and it has an update date timestamp.